### PR TITLE
Updated based on table given to us by Rebecca Boyd and her student

### PR DESCRIPTION
**Pull request by jmtjendra**

### CODE REVIEW

**Comment by jmtjendra** [comment]
Oh, I should mention that it includes the AED images. Once pulled in, please verify that image URL is working.

File: ExamSchedule/1161ExamSchedule.csv [hunk 1]
```diff
@@ -0,0 +1,903 @@
+Exam,Section(s),Day,Date,Start,Finish,Location,,,,,,,,
```
Comment (zachomedia) [inline ExamSchedule/1161ExamSchedule.csv]: @jmtjendra I don't think this is supposed to be here.. Given that the Winter 2016 exam schedule isn't out yet and these are Fall 2015 exam dates.

**Comment by jmtjendra** [comment]
@zachomedia You are correct - all I want is my commits beginning "Images of AEDs on campus", so 2c1ecd9, 54f16ac, 17f865f. Is there a way for you to merge just these 3 commits?

**Comment by jmtjendra** [comment]
Abandoning pull request. Fixes did not go as expected...

**Pull request closed without merging**